pkg/util/k8sutil/k8sutil.go

```diff
@@ -213,26 +260,30 @@ func CreatePeerService(ctx context.Context, kubecli kubernetes.Interface, cluste
 		TargetPort: intstr.FromInt(2380),
 		Protocol:   v1.ProtocolTCP,
 	}}
-
-	service := &api.ServicePolicy{
-		Type:      v1.ServiceTypeClusterIP,
-		ClusterIP: v1.ClusterIPNone,
+	
+	service := &api.ServicePolicy{}
+	annotations := map[string]string{}
```

Comment:
Same here, probably you could re-use the function adding a couple of parameters

Reply:
this one is actually unnecessary, fixing